tox.ini

```diff
@@ -0,0 +1,16 @@
+[tox]
+envlist =
+  py{27,34,35,36,37,py,py3}-unit,
+  py{27,34,35,36,37,py,py3}-lint,
+skip_missing_interpreters = True
```

Comment:
We're not supporting Python 3 (yet) or PyPy (yet?), so there's no reason to run or care for tests for those. Wouldn't it be better to just not list them rather than ignore them if non-existent?

Reply:
On travis this are not run because only the environment is set up for python 2.7.

For local dev the tests never get tun on pypy nor py3 because the package is dfefined as a python 2.7 only package.

I'll remove the line, I don't think it hurts knowing this will be useful for the porting effort.